tools/pipelines/templates/include-test-stability.yml

```diff
@@ -129,24 +111,11 @@ jobs:
         displayName: Start Test
 
       - ${{ each taskTestStep in parameters.taskTest }}:
-        # Test - With coverage
-        - ${{ if and(eq(variables['testCoverage'], true), startsWith(taskTestStep, 'ci:test')) }}:
-          - task: Npm@1
-            displayName: npm run ${{ taskTestStep }}:coverage
-            inputs:
-              command: 'custom'
-              workingDir: ${{ parameters.buildDirectory }}
-              customCommand: 'run ${{ taskTestStep }}:coverage'
-            condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
-        # Test - No coverage
-        - ${{ if or(eq(variables['testCoverage'], false), eq(startsWith(taskTestStep, 'ci:test'), false)) }}:
-            - task: Npm@1
-              displayName: npm run ${{ taskTestStep }}
-              inputs:
-                command: 'custom'
-                workingDir: ${{ parameters.buildDirectory }}
-                customCommand: 'run ${{ taskTestStep }}'
-              condition: and(succeededOrFailed(), eq(variables['startTest'], 'true'))
+        - template: include-task-test.yml
```

Comment:
Have you tested this? The file is named include-test-task.yml in this PR, although it was include-task-test.yml in the previous, reverted PR.

Reply:
Oh my bad, I initially tested it in my personal branch. I intended to modify the file name when moving it to the PR, but I did change it in the `build-npm-package'`` but forgot to make the corresponding change in this one. Have fixed it